bot/config.py.mynewt.sample

```diff
@@ -102,6 +102,15 @@ m['iut_config'] = {
         ],
     },
 
+      "mtu_400.conf": {
+        "overlay": {
+            'BTTESTER_L2CAP_COC_MTU': '400',
```

Comment:
Can we change MTU for all test cases or is this one special?

Reply:
I investigated this test a little more and there's an issue of how the WID describes what should be done. This test checks if credits are increasing when IUT sends SDUs of size equal to peer (PTS) MPS. The wording of WID is: "Upper Tester command IUT to send at least 4 frames of LE data packets to the PTS." It caused confusion, same as in PTS CASE0070320, that l2cap_send_data should be called with data of size 4*peer_mps, but in fact, we just need to send SDU of size peer_mps. Then, PTS will ask for sending more such SDUs, and will ask it 3 more times.

What's more, PTS doesn't follow MPS value from IXIT in this test. In conclusion, MTU might stay as it is, I will change WID 57 and test will pass